[Scientific Linux] build error: flow_control.c missing limits.h include?

Please answer these questions before submitting your issue. 
 
### Should this be an issue in the gRPC issue tracker?

Yes (but it's minor)
 
### What version of gRPC and what language are you using?
 version: 1.6.x
lang: C/C++
 
### What operating system (Linux, Windows, …) and version?
 Scientific Linux 7.3 
 
### What runtime / compiler are you using (e.g. python version or version of gcc)
 gcc version 4.8.5 20150623 (Red Hat 4.8.5-11) (GCC)
 
### What did you do?
Installed "Development Tools" for Scientific Linux i.e.
```
yum groupinstall "Development Tools"
```
plus 'autoconf' and 'libtool'

Followed the "Build from Source" instructions in the 'Install.md' document, error occurred during 'make' command.
 
### What did you expect to see?
 Clean build, no errors
 
### What did you see instead?
```
> make
[MAKE]    Generating cache.mk
...
[C]       Compiling src/core/ext/transport/chttp2/transport/flow_control.c
In file included from ./src/core/lib/transport/metadata.h:24:0,
                 from ./src/core/lib/transport/metadata_batch.h:28,
                 from ./src/core/lib/transport/transport.h:31,
                 from ./src/core/ext/transport/chttp2/transport/frame_data.h:29,
                 from ./src/core/ext/transport/chttp2/transport/internal.h:26,
                 from src/core/ext/transport/chttp2/transport/flow_control.c:19:
src/core/ext/transport/chttp2/transport/flow_control.c: In function ‘grpc_chttp2_flowctl_get_bdp_action’:
src/core/ext/transport/chttp2/transport/flow_control.c:486:49: error: ‘INT_MAX’ undeclared (first use in this function)
           GPR_MAX((int32_t)GPR_CLAMP(bw_dbl, 0, INT_MAX) / 1000, bdp), 16384,
                                                 ^
include/grpc/support/useful.h:26:34: note: in definition of macro ‘GPR_CLAMP’
 #define GPR_CLAMP(a, min, max) ((a) < (min) ? (min) : (a) > (max) ? (max) : (a))
                                  ^
src/core/ext/transport/chttp2/transport/flow_control.c:486:11: note: in expansion of macro ‘GPR_MAX’
           GPR_MAX((int32_t)GPR_CLAMP(bw_dbl, 0, INT_MAX) / 1000, bdp), 16384,
           ^
src/core/ext/transport/chttp2/transport/flow_control.c:486:28: note: in expansion of macro ‘GPR_CLAMP’
           GPR_MAX((int32_t)GPR_CLAMP(bw_dbl, 0, INT_MAX) / 1000, bdp), 16384,
                            ^
src/core/ext/transport/chttp2/transport/flow_control.c:486:49: note: each undeclared identifier is reported only once for each function it appears in
           GPR_MAX((int32_t)GPR_CLAMP(bw_dbl, 0, INT_MAX) / 1000, bdp), 16384,
                                                 ^
include/grpc/support/useful.h:26:34: note: in definition of macro ‘GPR_CLAMP’
 #define GPR_CLAMP(a, min, max) ((a) < (min) ? (min) : (a) > (max) ? (max) : (a))
                                  ^
src/core/ext/transport/chttp2/transport/flow_control.c:486:11: note: in expansion of macro ‘GPR_MAX’
           GPR_MAX((int32_t)GPR_CLAMP(bw_dbl, 0, INT_MAX) / 1000, bdp), 16384,
           ^
src/core/ext/transport/chttp2/transport/flow_control.c:486:28: note: in expansion of macro ‘GPR_CLAMP’
           GPR_MAX((int32_t)GPR_CLAMP(bw_dbl, 0, INT_MAX) / 1000, bdp), 16384,
                            ^
make: *** [/home/otcx/grpc/objs/opt/src/core/ext/transport/chttp2/transport/flow_control.o] Error 1
```

The C source file prior to `flow_control.c`, namely  `chttp2_transport.c`, which also uses INT_MAX,  compiled okay, though it has `limits.h` as an inclusion. Manually adding `limits.h` to `flow_control.c` allows compilation.

### Anything else we should know about your project / environment?
I explicitly export the `PKG_CONFIG_PATH` environment variable.

The same gRPC version build is successful on OpenSuse 13.2 without any intervention; gcc version 4.8.3 20140627 [gcc-4_8-branch revision 212064] (SUSE Linux), gRPC version 1.6.x.

--- a/src/core/ext/transport/chttp2/transport/flow_control.c
+++ b/src/core/ext/transport/chttp2/transport/flow_control.c
@@ -18,6 +18,7 @@
 
 #include "src/core/ext/transport/chttp2/transport/internal.h"
 
+#include <limits.h>
 #include <math.h>
 #include <string.h>
 

